website/src/documentation/sdks/python-dependencies.md

```diff
@@ -26,460 +26,41 @@ behavior in the service. If you are using any of these packages in your code, be
 
-</details>
+You can also retrieve the dependency list from the command line using the following process:
 
-<details><summary markdown="span"><b>2.16.0</b></summary>
+1.  Create a clean virtual environment on your local machine.
 
-<p>Beam SDK for Python 2.16.0 has the following compile and
-  runtime dependencies.</p>
+    Python 3:
 
-<table class="table-bordered table-striped">
-  <tr><th>Package</th><th>Version</th></tr>
-  <tr><td>avro-python3</td><td>&gt;=1.8.1,&lt;2.0.0; python_version &gt;= "3.0"</td></tr>
-  <tr><td>avro</td><td>&gt;=1.8.1,&lt;2.0.0; python_version &lt; "3.0"</td></tr>
-  <tr><td>cachetools</td><td>&gt;=3.1.0,&lt;4</td></tr>
-  <tr><td>crcmod</td><td>&gt;=1.7,&lt;2.0</td></tr>
-  <tr><td>dill</td><td>&gt;=0.3.0,&lt;0.3.1</td></tr>
-  <tr><td>fastavro</td><td>&gt;=0.21.4,&lt;0.22</td></tr>
-  <tr><td>funcsigs</td><td>&gt;=1.0.2,&lt;2; python_version &lt; "3.0"</td></tr>
-  <tr><td>future</td><td>&gt;=0.16.0,&lt;1.0.0</td></tr>
-  <tr><td>futures</td><td>&gt;=3.2.0,&lt;4.0.0; python_version &lt; "3.0"</td></tr>
-  <tr><td>google-apitools</td><td>&gt;=0.5.28,&lt;0.5.29</td></tr>
-  <tr><td>google-cloud-bigquery</td><td>&gt;=1.6.0,&lt;1.18.0</td></tr>
-  <tr><td>google-cloud-bigtable</td><td>&gt;=0.31.1,&lt;1.1.0</td></tr>
-  <tr><td>google-cloud-core</td><td>&gt;=0.28.1,&lt;2</td></tr>
-  <tr><td>google-cloud-datastore</td><td>&gt;=1.7.1,&lt;1.8.0</td></tr>
-  <tr><td>google-cloud-pubsub</td><td>&gt;=0.39.0,&lt;1.1.0</td></tr>
-  <tr><td>googledatastore</td><td>&gt;=7.0.1,&lt;7.1; python_version &lt; "3.0"</td></tr>
-  <tr><td>grpcio</td><td>&gt;=1.12.1,&lt;2</td></tr>
-  <tr><td>hdfs</td><td>&gt;=2.1.0,&lt;3.0.0</td></tr>
-  <tr><td>httplib2</td><td>&gt;=0.8,&lt;=0.12.0</td></tr>
-  <tr><td>mock</td><td>&gt;=1.0.1,&lt;3.0.0</td></tr>
-  <tr><td>oauth2client</td><td>&gt;=2.0.1,&lt;4</td></tr>
-  <tr><td>proto-google-cloud-datastore-v1</td><td>&gt;=0.90.0,&lt;=0.90.4; python_version &lt; "3.0"</td></tr>
-  <tr><td>protobuf</td><td>&gt;=3.5.0.post1,&lt;4</td></tr>
-  <tr><td>pyarrow</td><td>&gt;=0.11.1,&lt;0.15.0; python_version &gt;= "3.0" or platform_system != "Windows"</td></tr>
-  <tr><td>pydot</td><td>&gt;=1.2.0,&lt;2</td></tr>
-  <tr><td>pymongo</td><td>&gt;=3.8.0,&lt;4.0.0</td></tr>
-  <tr><td>python-dateutil</td><td>&gt;=2.8.0,&lt;3</td></tr>
-  <tr><td>pytz</td><td>&gt;=2018.3</td></tr>
-  <tr><td>pyvcf</td><td>&gt;=0.6.8,&lt;0.7.0; python_version &lt; "3.0"</td></tr>
-  <tr><td>pyyaml</td><td>&gt;=3.12,&lt;4.0.0</td></tr>
-  <tr><td>typing</td><td>&gt;=3.6.0,&lt;3.7.0; python_version &lt; "3.5.0"</td></tr>
-</table>
+    ```
+    $ python3 -m venv env && source env/bin/activate
+    ```
+    
+    Python 2: 
 
-</details>
+    ```
+    $ pip install virtualenv && virtualenv env && source env/bin/activate
+    ```
 
-<details><summary markdown="span"><b>2.15.0</b></summary>
+2. [Install the Beam Python SDK]({{ site.baseurl }}/get-started/quickstart-py/#download-and-install).
```

Comment:
To not have to jump to an additional step, can we just put the command to run?

```
pip install -U apache-beam==2.18.0
```

Reply:
I actually asked in an earlier comment to link existing installation instruction to keep all things related to installation in one place, and not duplicate these instructions on multiple pages. For example https://beam.apache.org/get-started/quickstart-py/#download-and-install mentions how to install beam with [gcp] deps, etc.

Reply:
Got it